WIP-CHANGELOG.md

```diff
@@ -1,80 +1,10 @@
-
 ### Added
 
-- Dex configuration to accept groups from Okta as an OIDC provider
-- Added record `cluster.name` in all logs to elasticsearch that matches the cluster setting `global.clusterName`
-- Role mapping from OIDC groups to roles in user grafana
-- Configuration options regarding resources/tolerations for prometheus-elasticsearch-exporter
-- Options to disable different types of backups.
-- Harbor image storage can now be set to `filesystem` in order to use persistent volumes instead of object storage.
-- Object storage is now optional. There is a new option to set object storage type to `none`. If you disable object storage, then you must also disable any feature that uses object storage (mostly all backups).
-- Two new InfluxDB users to used by prometheus for writing metrics to InfluxDB.
-- Multicluster support for some dashboards in Grafana.
-- More config options for falco sidekick (tolerations, resources, affinity, and nodeSelector)
-- Option to configure serviceMonitor for elasticsearch exporter
-- Option to add more redirect URIs for the `kubelogin` client in dex.
-- Option to disable the creation of user namespaces (RBAC will still be created)
-- The possibility to configure resources, affinity, tolerations, and nodeSelector for all Harbor pods.
-
 ### Changed
-
-- Updated user grafana chart to 6.1.11 and app version to 7.3.3
-- The `stable/elasticsearch-exporter` helm chart has been replaced by `prometheus-community/prometheus-elasticsearch-exporter`
-- OIDC group claims added to Harbor
-- The options `s3` and `gcs` for `harbor.persistence.type` have been replaced with `objectStorage` and will then match the type set in the global object storage configuration.
-- Bump kubectl to v1.18.13
-- InfluxDB is now exposed via ingress.
-- Prometheus in workload cluster now pushes metrics directly to InfluxDB.
-- The prometheus release `wc-scraper` has been renamed to `wc-reader`.
-  Now wc-reader only reads from the workload_cluster database in InfluxDB.
-- InfluxDB helm chart upgraded to `4.8.11`.
-- `kube-prometheus-stack` updated to version `12.8.0`.
-- Bump prometheus to `2.23.0`.
-- Added example config for Kibana group mapping from an OIDC provider
-- Replaced `kiwigrid/fluentd-elasticsearch` helm chart with `kokuwa/fluentd-elasticsearch`.
-- Replaced `stable/fluentd` helm chart with `bitnami/fluentd`.
-- StorageClasses are now enabled/disabled in the `{wc,sc}-cofig.yaml` files.
-- Mount path and IP/hostname is now configurable in `nfs-client-provisioner`.
-- Upgraded `cert-manager` to `1.1.0`.
-- Moved the `bootstrap/letsencrypt` helm chart to the apps step and renamed it to `issuers`.
-  The issuers are now installed after cert-manager.
-  You can now select which namespaces to install the letsencrypt issuers.
-- Helm upgraded to `v3.5.0`.
-- InfluxDB upgraded to `v4.8.12`.
-- Resource requests/limits have been updated for all Harbor pods.
 - The pipeline now stops on lint errors.
```

Comment:
This should be remove as it is about the pipeline, right?

Reply:
Well it is about the scripts in this repo, but it isn't a user facing change so I don't think it is needed here. Fine to remove. :slightly_smiling_face: